enterprise/frontend/src/embedding-sdk/test/storybook-themes.ts

```diff
@@ -249,8 +249,8 @@ const proficiencyTheme: MetabaseTheme = {
     filter: proficiencyColors.primary,
     "text-primary": proficiencyColors.lightGrey,
     "text-secondary": proficiencyColors.lightGrey,
-    "text-tertiary": "rgba(0, 0, 0, 0.4)",
-    border: "rgba(0, 0, 0, 0.12)",
+    "text-tertiary": "#979898",
+    border: "#DEDFDF",
```

Comment:
Use non-alpha values for now. We should update the docs to reflect that this should be a `#RRGGBB` value for best results.